build.sbt

```diff
@@ -217,3 +217,10 @@ lazy val firechip = conditionalDependsOn(project in file("generators/firechip"))
     testGrouping in Test := isolateAllTests( (definedTests in Test).value ),
     testOptions in Test += Tests.Argument("-oF")
   )
+lazy val fpgaShells = (project in file("./fpga/fpga-shells"))
+  .dependsOn(rocketchip, sifive_blocks)
+  .settings(commonSettings)
+
+lazy val freedomPlatforms = (project in file("./fpga"))
```

Comment:
I think we should just call it `fpgaPlatforms` or `fpgaPrototyping`. We're going to rapidly evolve away from the original (unmaintained) sources.